parkerbane/MailBux
# #168968634 Configure Eslint

#### What does this PR do?
Implements the `Airbnb eslint` style guide for producing clean and well-formatted code.

#### How should this be manually tested?
The bash commands `$ npm run lint` and `$ npm test` will run the eslint module and should output no errors if the code style does not break any of the rules contained in the `.eslintrc.json` file.

#### What are the relevant pivotal tracker stories?
[#168968634](https://www.pivotaltracker.com/story/show/168968634)
